src/main/kotlin/com/netflix/dgs/plugin/hints/DgsInputArgumentValidationInspector.kt

```diff
@@ -124,7 +124,7 @@ class DgsInputArgumentValidationInspector : AbstractBaseUastLocalInspectionTool(
             val expectedInputArgumentHint = InputArgumentUtils.getHintForInputArgument(graphQLInput, typeDefinitionRegistry, isJavaFile)
             if (expectedInputArgumentHint.contains("collectionType")) {
                 val expectedCollectionType = InputArgumentUtils.getCollectionType(graphQLInput.type!!, isJavaFile)
-                val inputCollectionType = inputArgumentAnnotation.findAttributeValue("collectionType")?.text?.removeSuffixIfPresent(".class")
+                val inputCollectionType = inputArgumentAnnotation.findAttributeValue("collectionType")?.text?.removeSuffixIfPresent(".class")?.removeSuffixIfPresent("::class")
```

Comment:
`::class` shouldn't be there in the first place, so this should probably be fixed elsewhere.